### ISSUE TITLE
Sketcher: Do not align the view to the sketch plane when editing

### ISSUE THREAD

**Issue by kaiwas** [FreeCAD/FreeCAD#12895]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Currently, when edit mode is activated, the camera is always oriented along the sketch plane.
But this is not always required. Sometimes it is more convenient to edit a sketch without changing this orientation.
For example, in the video there are two walls that are trimmed with external geometry.
In the first case, the camera switches to a vertical view and to search for external geometry you have to click on an edge (aim).
In the second (if the camera did not change position) - the external geometry can be taken from the wall plane (easier to hit). In addition, this would allow us to avoid losing objects from the composition, because the view would remain in a fixed position.
If you often enter and exit the editing mode, then due to constant rotations or changes in the position of the view, focus on objects is lost.

https://github.com/FreeCAD/FreeCAD/assets/32273833/30e11cc6-375c-4992-85bd-4debfd0b2b62

In my opinion, it would be possible to implement an option in the settings that would allow you to disable auto-rotation of the camera when editing a sketch if necessary. (on the picture).
I assume that implementing such an option is not a difficult task.
![Not align](https://github.com/FreeCAD/FreeCAD/assets/32273833/3e7dbf05-45fb-4a50-8d7b-bf9ddf08790e)

### Full version info

```shell
Any version
```

### Subproject(s) affected?

Sketcher

### Anything else?

Note
In the video (with the second wall), the camera in edit mode does not match the view in normal mode. Because I wasn't looking for the camera position very accurately. But the important principle here is that moving to the sketch plane is not always an urgent need. Sometimes it is more convenient to edit a sketch “as is”.
In addition, it is always possible to select the orientation using the navigation cube (if we are talking about standard planes)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
https://forum.freecad.org/viewtopic.php?f=8&t=31000

**Comment by kaiwas** [comment]
> https://forum.freecad.org/viewtopic.php?f=8&t=31000

I don’t quite understand what is being discussed in the topic linked to. It looks like this is part design WB behavior. But this GitHub request has nothing to do with part design WB.

**Comment by kaiwas** [comment]
General meaning
![Image](https://github.com/user-attachments/assets/5aab7291-be72-4ec7-b3ad-58320f69bb7e)

**Comment by eduncan911** [comment]
I could have sworn I filed a feature request for exactly this in the release candidates of 1.0.  But I can't find it...  @kaiwas nailed it in the screenshot above.

FreeCAD LinkBranch calls it "**Auto adjust camera on editing**" within the Sketch properties.

The default behavior is extremely disorienting, **_especially for disabled people_**.

It's also the top complain I have to walk people around every time I introduce them to FC in person (getting them to switch from other professional ones - that all have this "feature").

I also like the suggesion in #16579 to toggle it ON/OFF from the Navigation Cube (which is also possibly a duplicate of this issue).

FreeCAD LinkBranch has had this "feature" for many years, and it was (and continues to be) a culture shock every time it happens in FC 1.x now.

<img width="800" height="788" alt="Image" src="https://github.com/user-attachments/assets/d13063f4-23a1-4d29-b6f5-d9fcf955b76e" />

---

In short: Don't frack with the camera view when opening/editing a sketch!  Or at least, add the option to disable this behavior.

**Comment by eduncan911** [comment]
@maxwxyz is there a way to prioritize this to help for disability reasons?

**Comment by maxwxyz** [comment]
there is already a draft PR by @hyarion maybe if you can charm him..

**Comment by eduncan911** [comment]
Ah, found it!  #20759

**Comment by eduncan911** [comment]
Related forum thread: https://forum.freecad.org/viewtopic.php?t=53017&sid=5d5712919cc86206494642c54747d44f

**Comment by kaiwas** [comment]
> Related forum thread: https://forum.freecad.org/viewtopic.php?t=53017&sid=5d5712919cc86206494642c54747d44f

dmacro seems to have decided for everyone that this is not necessary)

**Comment by GeneGH** [comment]
> dmacro seems to have decided for everyone that this is not necessary)

That is completely unfair. Drmacro very clearly pointed out that his opinion was not binding on anyone else.

**Comment by ledgeronix** [comment]
Has this been addressed? Is there a toggle I can select to turn off the change camera?

**Comment by CaribouFrequency** [comment]
Another manifestation of the same behaviour is that not only does it align the view to the sketch plane, but it will often reorient the sketch, rotating it about one or more axes. This is a bit of a nuisance, as I have to readjust my view after opening a sketch about 80% of the time. 

I have never really had to think about it before, but my workflow is to adjust the view to the desired position _then_ edit the sketch. I feel like that makes sense, because you may finish one task, rotate to the other side of your model to do some work there, make a sketch, and begin that task. The program making assumptions on my behalf interferes with that process. I'm coming from a Fusion background, and aside from the expected differences in how the two programs operate, this is the biggest headache I have encountered so far.
